fpgaconvnet/models/partition/update.py

```diff
@@ -40,6 +40,21 @@ def update(self):
         if self.graph.nodes[node]["type"] in MULTIPORT_LAYERS_IN:
             self.update_multiport_buffer_depth(node)
 
+def update_io_ports(self):
```

Comment:
where is this function called from? I only found it at "https://github.com/AlexMontgomerie/fpgaconvnet-optimiser/blob/c49f0f5ed6063158c68d799e7652efe162746a3f/fpgaconvnet/optimiser/solvers/solver.py#L350" and it is commented

Reply:
That was an initial attempt to deal with the multiport layers.
This one is not currently being called anywhere.
I kept it there just in case we needed it in the future, but I don't think this is happening.
I am going to completely remove this one